server/id/id.go

```diff
@@ -55,7 +55,7 @@ type allocatorImpl struct {
 }
 
 // NewAllocator creates a new ID Allocator.
-func NewAllocator(client *clientv3.Client, rootPath, allocPath, label, member string) Allocator {
+func NewAllocator(client *clientv3.Client, rootPath string, allocPath string, label string, member string) Allocator {
```

Comment:
why you want to change arguments to specify identical type for each individual argument?

Reply:
It was suggested to stay consistent with the style https://github.com/tikv/pd/pull/5284#discussion_r915597292